Dockerfile

```diff
@@ -13,10 +13,15 @@ COPY pom.xml /home/gis
 
 RUN mvn -f /home/gis/pom.xml clean package -Dmaven.wagon.http.ssl.allowall=true -Dmaven.wagon.http.ssl.insecure=true
 
+# the two lines below can't be used in the github workflow because the files don't exist
```

Comment:
do we want to keep these comments?

Reply:
They're useful when building the app locally because it's easier than using build args from the command line. They're not strictly necessary though, do you think it would be better to leave them out?

Reply:
Oh ok, got it. Can we indicate that in the comments like which lines to use for local dev and which lines to comment

Reply:
Yep, done

Reply:
👍